File: C_Programs/MultiplicationTable.cpp
//Write a program in C to display the multiplication table of a given integer.

#include <stdio.h>
int main()
{
   int j,n;
   printf("Input any number : ");
   scanf("%d",&n);
   printf("\n");
   for(j=1;j<=10;j++)
   {
     printf("%d X %d = %d \n",n,j,n*j);
   }
   return 0;
} 

File: Basic_Java_Programs/MultiplicationTable.java
//Write a program in C to display the multiplication table of a given integer.
import java.util.*;

public class MultiplicationTable {
    public static void main(String[] args)
    {
        int j,n;
        Scanner sc=new Scanner(System.in);

        System.out.println("Input any number : ");
        n=sc.nextInt();
        System.out.println();
        for(j=1;j<=10;j++)
        {
            System.out.println(n+" X "+j+" = "+n*j );
        }
    }
}
